src/cryptonote_basic/account.h

```diff
@@ -44,18 +44,27 @@ namespace cryptonote
     crypto::secret_key   m_view_secret_key;
     std::vector<crypto::secret_key> m_multisig_keys;
     hw::device *m_device = &hw::get_device("default");
+    crypto::chacha_iv m_encryption_iv;
 
     BEGIN_KV_SERIALIZE_MAP()
       KV_SERIALIZE(m_account_address)
       KV_SERIALIZE_VAL_POD_AS_BLOB_FORCE(m_spend_secret_key)
       KV_SERIALIZE_VAL_POD_AS_BLOB_FORCE(m_view_secret_key)
       KV_SERIALIZE_CONTAINER_POD_AS_BLOB(m_multisig_keys)
+      const crypto::chacha_iv default_iv{{0, 0, 0, 0, 0, 0, 0, 0}};
+      KV_SERIALIZE_VAL_POD_AS_BLOB_OPT(m_encryption_iv, default_iv)
```

Comment:
Bikeshedding: how about using `boost::value_initialized`?
```c++
KV_SERIALIZE_VAL_POD_AS_BLOB_OPT(m_encryption_iv, (crypto::chacha_iv)boost::value_initialized<crypto::chacha_iv>())
```

Reply:
I want a known value here (zeroes here).

Reply:
`boost::value_initialized` does exactly that in this situation (initializes to zeroes). However, `boost::value_initialized` is more or less deprecated in C++11. The expression `const crypto::chacha_iv default_iv{}` performs value initialization (zeroes) or calls the default constructor if specified.

Reply:
In fact, technically, I don't care about the values, since they'll be overwritten if not present in the saved data.